.eslintrc

```diff
@@ -21,6 +19,7 @@
 			{
 				"argsIgnorePattern": "^_"
 			}
-		]
+		],
+		"@typescript-eslint/no-explicit-any": "off"
```

Comment:
I think we should consider turning this on as it is a TS best practice. I can think of one place in the code right now that has an explicit `any` and that would be the sessionize type added to the Window, are there other places that currently exist that are explicitly typed as `any`?